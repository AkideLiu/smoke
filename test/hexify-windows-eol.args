--command=ruby fixtures/hexify.rb
fixtures/hexify
