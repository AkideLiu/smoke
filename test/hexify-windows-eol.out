character
  succeeded
multiple-lines
  succeeded
sentence
  succeeded
word
  succeeded

4 tests, 0 failures
